tox.ini

```diff
@@ -4,4 +4,4 @@ envlist = flake8-py2.7
 
 [testenv:flake8-py2.7]
 deps = flake8
-commands = flake8 main.py appengine_config.py
+commands = flake8 phase1/main.py phase2/main.py phase3/main.py
```

Comment:
I don't like what I did. I put all the files as flake8 arguments, but there should be a better way.
Can you use something like?

```
commands = flake8 --exclude=lib
```